cinema/views.py

```diff
@@ -56,3 +56,14 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.all()
+    serializer_class = OrderSerializer
+
+    def get_queryset(self):
+        return self.queryset.filter(user=self.request.user)
```

Comment:
In the `get_queryset` method, it's better to directly use `Order.objects.filter(user=self.request.user)` instead of `self.queryset.filter(user=self.request.user)` to avoid potential issues with queryset caching or modifications.